npm/vite-dev-server/cypress/e2e/vite-dev-server.cy.ts

```diff
@@ -48,4 +47,15 @@ describe('Config options', () => {
     cy.contains(specWithWhitespace).click()
     cy.get('.passed > .num').should('contain', 1)
   })
+
+  it('supports @cypress/vite-dev-server', () => {
```

Comment:
What does this test actually prove works?

Reply:
The system test is hooked up the use `import { devServer } from '@cypress/vite-dev-server` via the `file:` package alias, so it's a test to prove that importing this as a standalone package and using the function signature works.